charts/jxboot-helmfile-resources/templates/jx-basic-auth-htpasswd-secret.yaml

```diff
@@ -1,4 +1,29 @@
 {{- if .Values.basicAuthSecrets.enabled }}
+apiVersion: kubernetes-client.io/v1
```

Comment:
I would suggest moving to ESO, https://external-secrets.io/v0.7.2/. https://github.com/external-secrets/kubernetes-external-secrets has been deprecated.

Reply:
i think there's still some work required on this front. https://github.com/jenkins-x-plugins/jx-secret/pull/387